packages/aws-cdk-lib/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts

```diff
@@ -405,11 +406,18 @@ export class AwsCustomResource extends Construct implements iam.IGrantable {
 
     this.props = props;
 
+    // Runtime regional fact should always return a known runtime string that lambda.Runtime
+    // can index off, but for type safety we also default it here.
+    const runtimeName = cdk.Stack.of(scope).regionalFact(FactName.DEFAULT_CR_NODE_VERSION);
+    const runtime = runtimeName
+      ? new lambda.Runtime(runtimeName, lambda.RuntimeFamily.NODEJS, { supportsInlineCode: true })
+      : lambda.Runtime.NODEJS_16_X;
```

Comment:
Yes, this. But then also for all other custom resources that we have, right?

So the route53 resolver and the s3 bucket deleter etc.?

Reply:
I got some but I think there are still more. Will do another pass.